src/Core/Analytics/Analytics.ts

```diff
@@ -0,0 +1,95 @@
+import postTelemetry from './post-telemetry';
+import collectId from './collect-id';
+import EventsQueue from './EventsQueue';
+import { Configuration } from '../configuration';
+import { CheckoutAttemptIdSession } from './types';
+
+export type AnalyticsProps = Pick<
+  Configuration,
+  'environment' | 'locale' | 'clientKey' | 'analytics' | 'amount'
+>;
+
+/**
+ * If the checkout attempt ID was stored more than fifteen minutes ago, then we should request a new ID.
+ * More here: COWEB-1099
+ */
+function confirmSessionDurationIsMaxFifteenMinutes(
+  checkoutAttemptIdSession: CheckoutAttemptIdSession
+): boolean {
+  if (!checkoutAttemptIdSession?.id) return false;
+
+  const fifteenMinInMs = 1000 * 60 * 15;
+  const fifteenMinAgoTimestamp = Date.now() - fifteenMinInMs;
+  return checkoutAttemptIdSession.timestamp > fifteenMinAgoTimestamp;
+}
```

Comment:
Not sure that you need this? Did you take a look into the COWEB ticket? It might be relevant only for web

Reply:
Fixed [776e262](https://github.com/Adyen/adyen-react-native/pull/230/commits/776e26244224590e86483fdda2680a8f572b1d04)